src/common/options/mgr.yaml.in

```diff
@@ -311,11 +311,13 @@ options:
 - name: mon_target_pg_per_osd
   type: uint
   level: advanced
-  desc: Automated PG management creates this many PGs per OSD
-  long_desc: When creating pools, the automated PG management logic will attempt to
-    reach this target.  In some circumstances, it may exceed this target, up to the
+  desc: Target number of PG replicas per OSD
+  long_desc: The PG autoscaler will arrange for approximately this number of PG
+    replicas on each OSD as shown by the PGS column of a ``ceph osd df``
```

Comment:
```suggestion
    on each OSD as shown by the PGS column of a ``ceph osd df``
```

Reply:
See above.  Do you feel strongly re me making clear that the math applies to replicas / shards not to PGs?  I welcome your thoughts.